ggshield/core/config/user_config.py

```diff
@@ -81,6 +83,25 @@ class IaCConfig(FilteredConfig):
     minimum_severity: str = "LOW"
 
 
+def validate_ghsa_id(value: str):
+    if not is_ghsa_valid(value):
+        raise ValidationError(
+            f"The given GHSA id {value} do not match the pattern '{GHSA_ID_PATTERN.pattern}'"
+        )
+
+
+@marshmallow_dataclass.dataclass
+class SCAIgnoredVulnerability(FilteredConfig):
+    """
+    A model of an ignored vulnerability for SCA
+    """
+
+    ghsa_id: str = field(metadata={"validate": validate_ghsa_id})
+    path: str
```

Comment:
What does the `path` represent here? Maybe add explanations in the docstring of `SCAIgnoredVulnerability`?